aiida/cmdline/commands/cmd_export.py

```diff
@@ -146,7 +152,7 @@ def create(output_file, codes, computers, groups, nodes, input_forward, create_r
 @options.FORCE(help='overwrite output file if it already exists')
 @options.SILENT()
 def migrate(input_file, output_file, force, silent, archive_format):
-    # pylint: disable=too-many-locals,too-many-statements,too-many-branches
+    # pylint: disable=too-many-locals,too-many-statements,too-many-branches,useless-import-alias
```

Comment:
indeed here instead of ignoring the warning we should do `from aiida.common import json` and remove this change (pylint ignore)

Reply:
Done